src/tests/Dashboard/container/Concepts.test.jsx

```diff
@@ -15,7 +16,7 @@ describe('Dashboard Concept Component', () => {
       concepts: [],
       isFetching: false,
     };
-    const wrapper = mount(<ConceptSearch {...props} />);
+    const wrapper = mount(<MemoryRouter><ConceptSearch {...props} /></MemoryRouter>);
```

Comment:
why did you have to change this file

Reply:
I made changes to the SideNavigation.jsx file to use the Link element hence I had to wrap the component being tested in the MemoryRouter

Reply:
Same here, you can only change for your link so as to stay focused to your ticket.

Reply:
it's part of the sideNavigation component which is rendered in all containers that are tested